### PR TITLE
PR for screencapture ref

### LINKED ISSUE #247: Create a reference page for screencapture

### PR DESCRIPTION
**Pull request by abaileyb**
# Advanced HW 7 submission

Closes #247 

Name:Aaron Bailey Berro
Uniqname: abberro

### CODE REVIEW

File: _commands/os_x/screencapture.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+~~~ bash
+#Takes a screenshot of the entire screen and saves it to the file screencap.png in the pwd
+screencapture screencap.png
+~~~
+
+<!--more-->
+
+### Where to install
+
+'screencapture' comes  built with OSX's command-line tools. If you don't have them, you can install them via:
```
Comment (mterwill) [inline _commands/os_x/screencapture.md]: You use \`backticks\` above, could you make that consistent here?

File: _commands/os_x/screencapture.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
+---
+---
+
+screencapture
+-------
+
+`screencapture` is used to capture an image of the whole, or part of the screen.
+
+~~~ bash
+#Takes a screenshot of the entire screen and saves it to the file screencap.png in the pwd
```
Comment (mterwill) [inline _commands/os_x/screencapture.md]: `pwd` stands for "print working directory". Saying 'cwd' would be more correct here imho.

**Comment by abaileyb** [comment]
fixed

**Comment by mterwill** [comment]
Thanks. This looks good to me; I'll merge + give you credit.

**Pull request merged**